client/src/keybind.c

```diff
@@ -126,6 +132,7 @@ void keybind_register(void)
 {
   app_registerKeybind(KEY_F, bind_fullscreen, NULL, "Full screen toggle");
   app_registerKeybind(KEY_V, bind_video     , NULL, "Video stream toggle");
+  app_registerKeybind(KEY_P, bind_showFPS   , NULL, "FPS display toggle");
```

Comment:
I think `KEY_D` would be a better option... `D` for Display.

Reply:
Changed to `KEY_D`. I have no strong feelings on the matter.